### DISCUSSION TITLE
Touch input is doubled on Steam Deck

### DISCUSSION THREAD

**Discussion by ALANVF** [ppy/osu#19484] [Q&A]
Touch input on the Steam Deck is doubled when playing lazer. It seems like both touch-down and touch-up events both trigger separate touch events. I verified that this is the issue by holding down for a short time before releasing, and they did both trigger touch events. I'm not sure if this is an issue with linux, or the Steam Deck hardware itself.

Note that this only happens when Raw Input is disabled. When enabled, touch input doesn't work very well at all.

Though I don't think it's necessary (seems to be a simple o!f problem), I can provide logs and/or video if needed.

**Comment by frenzibyte** [comment]
If toggling "Raw input" affects touch input, then that's pretty much not touch input, that's mouse input translated from touch. I don't own Steam Deck however so I can't investigate this any further.
